src/skip-to-content/skip-to-content.scss

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-skip-to-content {
+  @include iui-blur($opacity: 3);
+  @include iui-focus($offset: 2px, $thickness: 2px);
+  @include themed {
+    color: t(iui-color-foreground-accessory);
+  }
+
+  border-radius: $iui-border-radius-huge;
+  text-decoration: none;
+  padding: ($iui-baseline * 0.75) $iui-m;
+  position: absolute;
```

Comment:
This might be better as `position: fixed` so that it's based on viewport rather than scroll position

Reply:
Oh, that is a good idea, I hadn't thought of that.

Reply:
Fixed